__tests__/database/migration/index.test.ts

```diff
@@ -5,7 +5,7 @@ describe('TESTING migration file', () => {
     expect(DROP_ALL_DB).toBeFalsy();
   });
 
-  it('TEST should check TARGET_MIGRATION_VERSION to 5', () => {
-    expect(TARGET_MIGRATION_VERSION).toBe(6);
+  it('TEST should check TARGET_MIGRATION_VERSION to 7', () => {
+    expect(TARGET_MIGRATION_VERSION).toBe(7);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test case is updated correctly to check for the new `TARGET_MIGRATION_VERSION` value of 7. However, please ensure that the migration script (`MigrationVersion7`) is implemented and tested properly.
```diff
-    expect(TARGET_MIGRATION_VERSION).toBe(6);
+    expect(TARGET_MIGRATION_VERSION).toBe(7);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->